File: model/src/main/java/com/apeiron/igor/enums/GameUrl.java
package com.apeiron.igor.enums;

public class GameUrl {

    public final static String sendChosePerson = "/game/ChosePerson";
    public final static String sendAttack = "/game/Attack";

    public final static String subscribeEnemyPlayerChosePerson = "/queue/game/EnemyPlayerChosePerson";
    public final static String subscribeEnemyPlayerStep = "/queue/game/EnemyPlayerStep";
    public final static String subscribeGameEnd = "/queue/game/GameEnd";
    public final static String subscribeStartGame =  "/queue/game/StartGame";
    public final static String subscribeYourStep = "/queue/game/YourStep";
    public final static String subscribeEnemyPlayerAttack =  "/queue/game/EnemyPlayerAttack";
}

File: web-client/src/main/js/igor-client/src/app/enum/game-url.ts
export class GameUrl {
  private static base = "/app/game";
  private static subscribe = "/user/queue/game";

  public static sendChosePerson: string = GameUrl.base + "/ChosePerson";
  public static sendAttack: string = GameUrl.base + "/Attack";

  public static subscribeEnemyPlayerChosePerson: string = GameUrl.subscribe + "/EnemyPlayerChosePerson";
  public static subscribeEnemyPlayerStep: string = GameUrl.subscribe + "/EnemyPlayerStep";
  public static subscribeGameEnd: string = GameUrl.subscribe + "/GameEnd";
  public static subscribeStartGame: string = GameUrl.subscribe + "/StartGame";
  public static subscribeYourStep: string = GameUrl.subscribe + "/YourStep";
  public static subscribeEnemyPlayerAttack: string = GameUrl.subscribe + "/EnemyPlayerAttack";
}
